dds/DdsDcps.mpc

```diff
@@ -81,14 +81,22 @@ project(OpenDDS_Dcps): core, coverage_optional, \
     ../rules.dds.GNU
   }
 
+  verbatim (gnuace, top, 1) {
+    cmake_dir_install_dest=$(DESTDIR)$(INSTALL_PREFIX)/share/cmake/OpenDDS
```

Comment:
Instead of a separate variable this could use GNU Make's function for dirname.

At this point I wonder if this should be its own script.